### ISSUE TITLE
Yoroi wallet ERG stucked on loading (error no memory)

### ISSUE THREAD

**Issue by bukum86** [Emurgo/yoroi-frontend#2097]
I have huge problem. After two months of usage my Yoroi wallet is stucked at loading. Error message is no more memory ... Unfortunately for me I only have wallet password but not 15 word phrase. Please help me, how can I unstuck wallet or back it up safely.

Chrome extension, version is 4.5.0

**Comment by bukum86** [comment]
I tried everything to free as much memory as possible (16GB RAM), from uninstalling all other extensions, starting PC in safe mode etc. but nothing helped. Large amount of coins are not accessible

**Comment by SebastienGllmt** [comment]
Which version of Yoroi are you using? For Chrome?

**Comment by bukum86** [comment]
Its 4.5.0 version, Chrome browser

**Comment by zuzunker** [comment]
> Yoroi wallet is stucked at loading. Error message is no more memory

Hi! Is it possible you could send a screenshot of this?

> Please help me, how can I unstuck wallet or back it up safely.

We will look into what exactly could possibly cause a memory issue like this and then figure out a way for you to get the fix without ruining all the data.

Right now the latest published version is 4.5.3, check plz if Chrome will be able to update the extension successfully.

**Comment by acmukas** [comment]
I have the same problem does anyone found a solution?

**Comment by zuzunker** [comment]
There's a new release being prepared now for version 4.6 , once this version will be available - please try it out again, because  it contains multiple Ergo-related fixes and improvements

**Comment by JoeSuavo** [comment]
I'm also having this exact issue, hopefully it will be fixed with 4.6 as I was also dumb enough to not save my seed phrase

**Comment by zuzunker** [comment]
The version 4.6 is release and now available in Chome and Firefix, check please if it works for you after updating

**Comment by JoeSuavo** [comment]
Still giving the same issue, I made sure to update in chrome to 4.6.0.

**Comment by JoeSuavo** [comment]
> The version 4.6 is release and now available in Chome and Firefix, check please if it works for you after updating

Still giving the same issue, I made sure to update in chrome to 4.6.0.

**Comment by acmukas** [comment]
Hi, the problem still persists with new version 4.6

**Comment by acmukas** [comment]
Okay so now i have the same problem with my second wallet. I use these wallets for mining at normally there apears couple of hundreds transactions in a week or so, and the more transactions there is in the wallet the slower it stars working untill i get this error code out of memory. Until this problem persist o gonna use different walltet.

**Comment by zuzunker** [comment]
In the next version we will try to make priority for the performance issues specifically

**Comment by JoeSuavo** [comment]
> In the next version we will try to make priority for the performance issues specifically

Is there any update on this? I see it was removed from the goals of the 4.7.0 and 4.8.0 milestones, and I am still unable to access my wallet. Very unfortunate.

**Comment by acmukas** [comment]
> > In the next version we will try to make priority for the performance issues specifically
> 
> Is there any update on this? I see it was removed from the goals of the 4.7.0 and 4.8.0 milestones, and I am still unable to access my wallet. Very unfortunate.

I still have the same problem and now and on my second wallet :), PROBLEM STILL PERSIST!!!

**Comment by JoeSuavo** [comment]
> > > In the next version we will try to make priority for the performance issues specifically
> > 
> > 
> > Is there any update on this? I see it was removed from the goals of the 4.7.0 and 4.8.0 milestones, and I am still unable to access my wallet. Very unfortunate.
> 
> I still have the same problem and now and on my second wallet :), PROBLEM STILL PERSIST!!!

Yeah, same, apparently this is not a suitable wallet for mining, would've liked to know that before mining to it for a month... RIP

**Comment by acmukas** [comment]
The diffrence on my second that have the same problem, wallet shows my balance, but then it loads it starts at 0 balance and 0 transactions and transactions goes up i think by 30 and my balance goes up by transactions count, until it reaches i think about 300 transactions and then it crashes (out of memory)

**Comment by zuzunker** [comment]
@acmukas @JoeSuavo , would that be possible for you to provide the addresses where you are receiving your mining payouts? We are in the process of completely reworking some of our backend APIs and the storage approaches, will be great if we can check your addresses to confirm the main suspected bottleneck reasons

**Comment by acmukas** [comment]
@vsubhuman this is wallet not crashed but i need to wait like 30mins until it loads i will try get other wallet adresses 9fqRah2gi6r3V9QKkzMcGDGyTNETMy7fARANaP36sJa2eAk1Az

**Comment by acmukas** [comment]
@vsubhuman The second wallet adress 9fCPqmZ4cDFc6prbeFYQNzzDpFngiVZbXSN9b55apDJ4JqrCEDm

**Comment by JoeSuavo** [comment]
> @acmukas @JoeSuavo , would that be possible for you to provide the addresses where you are receiving your mining payouts? We are in the process of completely reworking some of our backend APIs and the storage approaches, will be great if we can check your addresses to confirm the main suspected bottleneck reasons

I wish I knew, but it stopped working so long ago that I don't remember the address or know how much is currently lost. Will be looking forward to a fix with bated breath

**Comment by jarstorm** [comment]
Hello, and thank you for your contribution.
We're currently reviewing long-standing issues to keep the tracker manageable. As this issue hasn't seen activity in some time, we're closing it.
If the problem persists or you'd like to follow up, feel free to comment and we'll reopen it.
